docs/source/auto_examples/index.rst

```diff
@@ -92,13 +92,13 @@ This is a set of runnable Jupyter notebooks.
 
   .. container:: sphx-glr-download
 
-    :download:`Download all examples in Python source code: auto_examples_python.zip <//Users/taugspurger/sandbox/dask-ml/docs/source/auto_examples/auto_examples_python.zip>`
+    :download:`Download all examples in Python source code: auto_examples_python.zip <//home/mrocklin/workspace/dask-ml/docs/source/auto_examples/auto_examples_python.zip>`
 
 
 
   .. container:: sphx-glr-download
 
-    :download:`Download all examples in Jupyter notebooks: auto_examples_jupyter.zip <//Users/taugspurger/sandbox/dask-ml/docs/source/auto_examples/auto_examples_jupyter.zip>`
+    :download:`Download all examples in Jupyter notebooks: auto_examples_jupyter.zip <//home/mrocklin/workspace/dask-ml/docs/source/auto_examples/auto_examples_jupyter.zip>`
```

Comment:
Hrm, this is an odd change.  This is an auto-generated file?  Presumably rtd will fill this in?

Reply:
Yeah... I'm not sure how but it seems to work.

I haven't found a way of doing these examples that I like. Some of the examples require too much memory / take too long for RTD to build them, so we have to generate them ahead of time.